cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -691,3 +691,80 @@ subjects:
   namespace: rook-ceph
 # OLM: END CMD REPORTER ROLEBINDING
 ---
+#
+# OLM: BEGIN CLUSTER ROLE
+---
+# Aspects for creation of monitoring resources
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-ceph-monitor
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+# OLM: END CLUSTER ROLE
+#
+# OLM: BEGIN CLUSTERROLE BINDING
+---
+# Allow creation of monitoring resources
+kind: ClusterRoleBinding
```

Comment:
I don't think we should be giving Rook blanket permissions to change monitoring in all namespaces. This should just be a RoleBinding which allows Rook to change monitoring in only the needed namespace.